tests/examples/CMakeLists.txt.in

```diff
@@ -27,6 +27,10 @@ foreach(_diff_file ${_diff_files})
     set(_destination "${CMAKE_CURRENT_SOURCE_DIR}/${_destination}")
     file(CREATE_LINK ${_file} ${_destination} SYMBOLIC)
   endforeach()
+
+  file(CREATE_LINK "${DEAL_II_SOURCE_DIR}/examples/${_step}"
+    "${CMAKE_CURRENT_SOURCE_DIR}/${_step}" SYMBOLIC
```

Comment:
This creates the link in the *source* directory (under `tests/`), right? I recognize that that is the same as for the files above, but that's awkward not the least because it makes things show up if you do `git status`. Wouldn't it be better if these links we created in the *build* directory?

Reply:
This is put inside the temporary source directory, which is inside the build directory. For me it is `./build/tests/examples/source/``.

Reply:
I see. I had assumed that `CMAKE_CURRENT_SOURCE_DIR` points into the source directory structure, not the build directory structure.

Reply:
@bangerth I take full responsibility for this elaborate CMake madness.